Geometry/CommonTopologies/interface/SimplePixelTopology.h

```diff
@@ -282,6 +286,33 @@ namespace phase2PixelTopology {
                                              9.0, 9.0, 8.0, 8.0, 8.0, 11.0, 9.0, 9.0, 9.0, 8.0, 8.0, 8.0, 11.0};
 }  // namespace phase2PixelTopology
 
+namespace phase1HIonPixelTopology {
+  using pixelTopology::phi0p09;
+
+  constexpr uint32_t maxNumClustersPerModules = 2048;
+
+  HOST_DEVICE_CONSTANT int16_t phicuts[phase1PixelTopology::nPairs]{phi0p09,
+                                                                    phi0p09,
+                                                                    phi0p09,
+                                                                    phi0p09,
+                                                                    phi0p09,
+                                                                    phi0p09,
+                                                                    phi0p09,
+                                                                    phi0p09,
+                                                                    phi0p09,
+                                                                    phi0p09,
+                                                                    phi0p09,
+                                                                    phi0p09,
+                                                                    phi0p09,
+                                                                    phi0p09,
+                                                                    phi0p09,
+                                                                    phi0p09,
+                                                                    phi0p09,
+                                                                    phi0p09,
+                                                                    phi0p09};
+
+}  // namespace phase1HIonPixelTopology
```

Comment:
what about all the other constants in the `phase1PixelTopology` ?

Reply:
All the other are the same and will be there through inheritance in the struct.

Reply:
Understood.

Can you add a comment in this regard, here and in the declaration of the struct where the inheritance happens ?